tests/pages/components/user-sidebar.js

```diff
@@ -0,0 +1,15 @@
+import { isVisible, text, isHidden, property } from 'ember-cli-page-object';
+
+export default {
+  scope : '.user-sidebar',
+
+  userSidebarCount    : isVisible(),
```

Comment:
I don't think we need this property. I would name it `userSidebarVisible` or `userSidebarRendered` in context, but if the removal of the "it renders" test above and the one assertion in the other test means we no longer use this property, just remove it from here.